src/fluid_ele/4C_fluid_ele_xwall_boundary.cpp

```diff
@@ -139,91 +138,18 @@ void Discret::Elements::FluidXWallBoundary::location_vector(const Core::FE::Disc
               lmowner.push_back(owner);
               lm.push_back(dof[j]);
             }
-
-            if (doDirichlet)
-            {
-              const std::vector<int>* flag = nullptr;
-              Core::Conditions::Condition* dirich = node->get_condition("Dirichlet");
-              if (dirich)
-              {
-                if (dirich->type() != Core::Conditions::PointDirichlet &&
-                    dirich->type() != Core::Conditions::LineDirichlet &&
-                    dirich->type() != Core::Conditions::SurfaceDirichlet &&
-                    dirich->type() != Core::Conditions::VolumeDirichlet)
-                  FOUR_C_THROW("condition with name Dirichlet is not of type Dirichlet");
-                flag = &dirich->parameters().get<std::vector<int>>("ONOFF");
-              }
-              for (unsigned j = 0; j < dof.size(); ++j)
-              {
-                if (flag && (*flag)[j])
-                  lmdirich.push_back(1);
-                else
-                  lmdirich.push_back(0);
-              }
-            }
           }
         }
 
         // fill the vector with element dofs
         //      const int owner = Owner();
         std::vector<int> dofx = dis.dof(dofset, this);
         if (dofx.size()) FOUR_C_THROW("no element dofs expected");
-        //      std::vector<int> dof;
-        //      if(dofx.size())
-        //      {
-        //        //only take the first four dofs (the real dofs, but not the enriched ones)
-        //        dof.push_back(dofx.at(0));
-        //        dof.push_back(dofx.at(1));
-        //        dof.push_back(dofx.at(2));
-        //        dof.push_back(dofx.at(3));
-        //      }
-
-        //      if (dof.size()) lmstride.push_back(dof.size());
-        //      for (unsigned j=0; j<dof.size(); ++j)
-        //      {
-        //        lmowner.push_back(owner);
-        //        lm.push_back(dof[j]);
-        //      }
+
 
         // fill the vector with face dofs
         if (this->num_dof_per_face(0) > 0)
           FOUR_C_THROW("set face_ from private to protected and uncomment");
```

Comment:
Probably not at the core of this PR, but the error message is wrong now, since the code to be uncommented is deleted.

@sebproell is this part of one of your recent changes?